collect_app/src/main/java/org/odk/collect/android/projects/ProjectListItemView.kt

```diff
@@ -25,18 +26,27 @@ class ProjectListItemView(context: Context, attrs: AttributeSet?) : FrameLayout(
     }
 
     private fun getSubtext(generalSettings: Settings): String {
-        val username = generalSettings.getString(GeneralKeys.KEY_USERNAME) ?: ""
-        var url = generalSettings.getString(GeneralKeys.KEY_SERVER_URL) ?: ""
+        val username = if (generalSettings.getString(GeneralKeys.KEY_PROTOCOL).equals(Collect.getInstance().getString(R.string.protocol_google_sheets))) {
+            generalSettings.getString(GeneralKeys.KEY_SELECTED_GOOGLE_ACCOUNT) ?: ""
```

Comment:
It actually looks like both these `:? ""` cases are untested to me. Could we add that to `ProjectLIstItemViewTest`?